lib/std/Build/Module.zig

```diff
@@ -84,16 +84,19 @@ pub const CSourceFiles = struct {
     /// else relative to the build root.
     files: []const []const u8,
     flags: []const []const u8,
+    precompiled_header: ?*Step.Compile = null,
```

Comment:
This should be a `LazyPath` rather than a `Step.Compile`. The pre-compiled header could come from any external process, or even committed directly to source control.